pkg/util/resource/resource.go

```diff
@@ -72,8 +72,19 @@ func MergeResourceListKeepMin(a, b corev1.ResourceList) corev1.ResourceList {
 // MergeResourceListKeepSum creates a new ResourceList holding all the values from a and b
 // and resolve potential conflicts by adding up the tow values.
 func MergeResourceListKeepSum(a, b corev1.ResourceList) corev1.ResourceList {
-	return mergeResourceList(a, b, func(a, b apiresource.Quantity) apiresource.Quantity {
+	return mergeResourceList(a, b, func(a, b resource.Quantity) resource.Quantity {
 		a.Add(b)
 		return a
 	})
 }
+
+// IsLessOrEqual returns true if for all the common keys the value in a is less than the
+// values in b
+func IsLessOrEqual(a, b corev1.ResourceList) bool {
```

Comment:
```suggestion
func RequestsLessThanOrEqualToLimits(r corev1.ResourceRequirements) bool {
```

Otherwise it doesn't make much sense to ignore values that are not listed in b.